test/src/080-tracing/main

```diff
@@ -0,0 +1,61 @@
+cvmfs_test_name="tracing"
+
+cvmfs_run_test() {
+    logfile=$1
+    sudo sh -c "echo \"export GLITE_VERSION='util'\" > /etc/cvmfs/config.d/grid.cern.ch.local"
+    repo="grid.cern.ch"
+    echo "Setup..."
+    rm -f /tmp/cvmfs-$repo.trace.log 2> /dev/null
+    echo "Mounting test repository..."
+    cvmfs_mount $repo "CVMFS_TRACEFILE=/tmp/cvmfs-@fqrn@.trace.log" "CVMFS_TRACEBUFFER=4096" "CVMFS_TRACEBUFFER_THRESHOLD=2048" > /dev/null
+
+    echo "Checking trace log creation..."
+    sudo cvmfs_talk tracebuffer flush > /dev/null
+
+    if [ ! -f /tmp/cvmfs-$repo.trace.log ]; then
+        echo "ERROR: Log file was not created"
+        return 2
+    fi
+
+    cp /dev/null /tmp/cvmfs-$repo.trace.log > /dev/null
+
+    echo "Executing a few operations to check tracing..."
+
+    ls /cvmfs/$repo/ > /dev/null
+    ls /cvmfs/$repo/vc/bin > /dev/null
+    cat /cvmfs/$repo/README > /dev/null
+    df -P /cvmfs/$repo/README > /dev/null
+    getfattr -d -m ".*" /cvmfs/$repo/README > /dev/null
```

Comment:
There is a helper function for getting extended attributes because it's different on Linux and Mac